synapse/storage/database.py

```diff
@@ -702,6 +702,7 @@ async def runInteraction(
         func: Callable[..., R],
         *args: Any,
         db_autocommit: bool = False,
+        isolation_level: Optional[int] = None,
```

Comment:
looks like this parameter isn't being passed into `runWithConnection`.

Reply:
I've opened https://github.com/matrix-org/synapse/pull/11847 to fix this.